safeeyes/safeeyes.py

```diff
@@ -51,7 +51,7 @@ class SafeEyes(Gtk.Application):
     def __init__(self, system_locale, config, cli_args):
         super().__init__(
             application_id="io.github.slgobinath.SafeEyes",
-            flags=Gio.ApplicationFlags.IS_SERVICE
+            flags=Gio.ApplicationFlags.DEFAULT_FLAGS
```

Comment:
Can you please explain why this is necessary?

Reply:
When I added the `IS_SERVICE` flag in #560, I was mostly guessing which one would be the right one.
Turns out I was wrong - the `IS_SERVICE` flag appears to be used when an application is mostly/only started over DBus, but never directly. It waits for 10 seconds until the DBus Activation Message, and automatically quits after - since that message never arrives in our case, `do_activate` also never gets called, so it just doesn't work.
It only happened to work before because we were using `Gtk.Application` wrong.